src/pytest_inmanta_lsm/resources/docker-compose-iso6.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  postgres:
+    container_name: ${INMANTA_LSM_CONTAINER_DB_HOSTNAME}
+    image: postgres:${INMANTA_LSM_CONTAINER_DB_VERSION}
+    environment:
+      POSTGRES_USER: inmanta
+      POSTGRES_PASSWORD: inmanta
+
+  inmanta-server:
+    image: ${INMANTA_LSM_CONTAINER_ORCHESTRATOR_IMAGE}
+    volumes:
+      - ${INMANTA_LSM_CONTAINER_LICENSE_FILE}:/etc/inmanta/license/com.inmanta.license
+      - ${INMANTA_LSM_CONTAINER_ENTITLEMENT_FILE}:/etc/inmanta/license/com.inmanta.jwe
+      - ./my-server-conf.cfg:/etc/inmanta/inmanta.cfg
+      - ./my-env-file:/etc/inmanta/env
+    command: "server --wait-for-host ${INMANTA_LSM_CONTAINER_DB_HOSTNAME} --wait-for-port 5432"
```

Comment:
It's not clear to me why this command changed, and why the `authorized_keys` file disappeared (I guess the second is a consequence of the former). And I don't have sufficient experience / context to review this part.

Reply:
I changed it because it is not required anymore, as of pytest-inmanta-lsm==3.3.0, for container orchestrators we deploy ourself, we don't ssh in, we "docker exec" in